docs/components/arm/_index.md

```diff
@@ -19,54 +19,20 @@ outputs:
 # SME: Peter L
 ---
 
-A robotic arm is a serial chain of joints and links, with a fixed end and an end effector end.
-Joints may rotate, translate, or both, while a link is a rigid connector between joints.
+Arms provide an API for linear motion planning, including self-collision prevention and obstacle avoidance.
```

Comment:
```suggestion
The arm component provides an API for linear motion planning, including self-collision prevention and obstacle avoidance.
```
I'd word it this way so we know we're talking about the Viam component instead of the concept of the hardware itself.

Also what does "linear" mean here? Should that come out?

Reply:
Clarifying information about components is really beyond the scope of the PR. I can do that where we know the answer but this is an already enormous PR, and getting into fixing content when this is really just reorganizing things, is beyond what I am willing to do here. 

I am guessing it is linear because you do not define waypoints, it is strictly from the current point to a new defined point.

Reply:
Okay. Finished reviewing with this in mind.
I think there are waypoints along the path though, and it's definitely not a straight line if there are obstacles in the way, and also, the end effector often sweeps in curves because the geometry of an arm often makes this more efficient. I'll follow up myself about the "linear" thing to check accuracy.